src/FirebirdSql.Data.FirebirdClient/FirebirdClient/FbErrorCodes.cs

```diff
@@ -0,0 +1,3008 @@
+﻿namespace FirebirdSql.Data.FirebirdClient;
+
+/// <summary>
+/// Error codes occuring in Firebird database management system.
+/// 
+/// Messages are taken from documentation at: https://ib-aid.com/download/docs/firebird-language-reference-2.5/fblangref25-appx02-sqlcodes.html
```

Comment:
Rather take it from Firebird sources. For completeness and copyright sake. Add also some simple steps how the file/data was created. Something rough like [this](https://github.com/FirebirdSQL/NETProvider/blob/master/src/FirebirdSql.Data.FirebirdClient/Common/TimeZoneMapping.cs#L24) is fine.

I would maybe even keep the "raw" naming like `arith_except` from https://github.com/FirebirdSQL/firebird/blob/65f0e8b4f1b346b436456bf74d954c2e8204076f/src/include/firebird/impl/msg/jrd.h#L2.

Maybe worth updating also `IscErrorMessages` and `SqlStateMapping`.